docker/Dockerfile.base

```diff
@@ -52,3 +59,12 @@ RUN python3.7 -m pip install --upgrade virtualenv>=20.14.0 pip>=22.1.0 \
     && curl -sS https://bootstrap.pypa.io/get-pip.py | python3.11 \
     && python3.11 -m pip install --upgrade virtualenv>=20.14.0 pip>=22.1.0 \
     && python3 -m pip install yq && python3 -m pip cache purge
+
+# code-server
+RUN curl -fsSL --compressed  https://code-server.dev/install.sh | sh
+
+# stable entrypoint
+WORKDIR /opt/starwhale.bin
+COPY external/starwhale-entrypoint.sh starwhale-entrypoint
```

Comment:
use another name for `starwhale-entrypoint.sh` or `sw-docker-entrypoint.sh`?

`sw-docker-entrypoint` and `starwhale-entrypoint` maybe lead to a bit confused.

Reply:
done